scene/resources/resource_format_text.cpp

```diff
@@ -281,7 +282,7 @@ Ref<PackedScene> ResourceLoaderText::_parse_node_tag(VariantParser::ResourcePars
 					if (error == ERR_FILE_MISSING_DEPENDENCIES) {
 						// Resource loading error, just skip it.
 					} else if (error != ERR_FILE_EOF) {
-						_printerr();
+						ERR_PRINT(vformat("Parse Error: %s. [Resource file %s:%d]", VariantUtilityFunctions::error_string(error), res_path, lines));
```

Comment:
```suggestion
						ERR_PRINT(vformat("Parse Error: %s. [Resource file %s:%d]", error_names[error], res_path, lines));
```

The global variable `error_names` is always available because `core/error/error_list.h` is included in `core/typedefs.h`. It might be conceptually better to use `VariantUtilityFunctions`, but it's barely used in the codebase, and `error_names` is used in a few places.